src/components/useAccessToken.ts

```diff
@@ -3,6 +3,7 @@ import { getAccessTokenAction, refreshAccessTokenAction } from '../actions.js';
 import { useAuth } from './authkit-provider.js';
 
 const TOKEN_EXPIRY_BUFFER_SECONDS = 60;
+const MIN_REFRESH_DELAY = 15_000; // minimum delay before refreshing token
```

Comment:
It's a little funky that we have these constants using different units. Might be helpful to pick one, or at least be more explicit about what units we're using in the name.